README.md

```diff
@@ -1,5 +1,4 @@
-> ⓘ INFO: The STACKIT Python SDK is in beta and in active development.
-
+> ⓘ IMPORTANT NOTE: This repository is deprecated. Please use [this](https://github.com/stackitcloud/stackit-sdk-python) repository instead.
```

Comment:
```suggestion
> [!NOTE]  
> This repository is deprecated as the `core` was moved to [`stackit-sdk-python`](https://github.com/stackitcloud/stackit-sdk-python). Please use that repository instead.
```

Reply:
We should also deprecate the repo in the settings

Reply:
> We should also deprecate the repo in the settings

Yes. But I think we can do that only after this PR is merged because it will prevent any changes in the repository from happening.